WcaOnRails/spec/controllers/registrations_controller_spec.rb

```diff
@@ -2,6 +2,19 @@
 require 'rails_helper'
 
 RSpec.describe RegistrationsController do
+  def stripe_token_id(options = {})
+    default_card = {
+      number: "4242424242424242",
+      exp_month: 12,
+      exp_year: Time.now.year + 1,
+      cvc: "314",
+    }
+    default_card.merge!(options)
+    Stripe::Token.create(
+      card: default_card,
+    ).id
+  end
```

Comment:
It feels more natural to me to define first what I'm going to use afterwards in the same file (but not a rubyist speaking!).

If you don't mind I'll let it this way :p